config/hub/manager/ramen_manager_config.yaml

```diff
@@ -12,4 +12,4 @@ leaderElection:
 ramenControllerType: dr-hub
 maxConcurrentReconciles: 50
 volSync:
-  destinationCopyMethod: LocalDirect
+  destinationCopyMethod: Direct
```

Comment:
are we again defaulting to Direct mode instead of LocalDirect?

Reply:
@prsurve Yes, this PR is reverting to using `Direct` as the default mode. This mode has remained unchanged since ODF 4.13. In addition to retaining this mode, we have also implemented support for a rollback on failover. As a result, this mode has proven to be robust, and we have further enhanced it to address the two critical BZs open against 4.14, and more. Most importantly, we no longer need to have the local extra copies. Moreover, this PR significantly simplifies the entire copy operation.